tests/syndication_tests/tests.py

```diff
@@ -178,6 +178,19 @@ def test_rss2_feed_guid_permalink_true(self):
                 item.getElementsByTagName('guid')[0].attributes.get(
                     'isPermaLink').value, "true")
 
+    def test_rss2_single_enclosure(self):
+        response = self.client.get('/syndication/rss2/single-enclosure/')
+        doc = minidom.parseString(response.content)
+        chan = doc.getElementsByTagName('rss')[0].getElementsByTagName('channel')[0]
+        items = chan.getElementsByTagName('item')
+        for item in items:
+            enclosures = item.getElementsByTagName('enclosure')
+            self.assertEqual(len(enclosures), 1)
+
+    def test_rss2_multiple_enclosures(self):
+        with self.assertRaises(ValueError):
```

Comment:
please use `assertRaisesMessage` to verify it's the `ValueError` we expect (also makes the test easier to read and debug).